Ci hypothesis test

pvilledieu opened the pull request:
Hi Florian, 

Here is a new set of slides that merge the Confidence intervals and hypothesis testing slides. 
It is in a new folder. Let me know if it is not convenient for you!

pvilledieu committed:
adding CI and hypothesis test in the same set of slides

pvilledieu committed:
add html file

gustavekenedi committed:
change intro slide + minor formatting changes

gustavekenedi committed:
adjustments + formatting

gustavekenedi committed:
last changes to formatting

gustavekenedi committed:
update README.md

gustavekenedi committed:
add pdf

gustavekenedi commented:
Hi Florian,

The PR is ready to be merged :)

Thanks!

gustavekenedi commented:
Actually they're not final yet, i saw a few issues. I'll make the changes later and will give a pdf to my students now.

Sorry for the inconvenience.

floswald commented:
ok no worries tell me when i should have a look

gustavekenedi committed:
numerous formatting changes and some structure changes

gustavekenedi committed:
updated pdf

gustavekenedi commented:
I'm quite happy with this new version, let us know what you think if you have time :)

gustavekenedi committed:
rm pdf in folder home

floswald commented on chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd:
```diff
@@ -0,0 +1,1088 @@
+## Today: Deeper dive into ***statistical inference*** <sup>1</sup>
+
+.footnote[
+[1]: This lecture is based on the wonderful [confidence interval](https://moderndive.com/8-confidence-intervals.html) and [hypothesis testing](https://moderndive.com/9-hypothesis-testing.html) chapters of [ModernDive](https://moderndive.com/)]
```
footnote overwrites last bullet points for me

gustavekenedi replied:
hmm on my end it doesn't. does it overwrite it completely?

floswald replied:
yes, cannot read it. and I have got a *really* big screen. I'd go for removing some bullets to be on the safe side.

floswald commented on chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd:
````diff
@@ -0,0 +1,1088 @@
+* Let's start by drawing one random sample of size $n = 50$ from our bowl.
+
+```{r, echo = FALSE}
+bowl <- read.csv("https://www.dropbox.com/s/qpjsk0rfgc0gx80/pasta.csv?dl=1")
````
you have this in the package included:
```R
> data("pasta_jar",package = "ScPoEconometrics")
> ls()
[1] "pasta_jar"
```

gustavekenedi replied:
this is actually a different pasta bowl dataset: the census of the pasta I had, I didn't know that the package came with your pasta_jar data.

floswald commented on chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd:
```diff
@@ -0,0 +1,1088 @@
+om = par("mar")
+lowtop = c(om[1],om[2],0.1,om[4])
+
+library(magrittr)
```
i think it's good practice to show people which packages they need to load on a slide such that the code works.

gustavekenedi replied:
i agree, we've inserted in the slides the packages they need `moderndive` and `infer`. the only one missing is `tidyverse` but i would assume that students know that it should be loaded by now? i'd be happy to add it to slide 7 if you want!

regarding code, we do not show the code on how to construct the null distribution because we judged it wasn't essential to understanding the intuition. but we could add it somewhere if you would like.

floswald replied:
just add all libraries you load, so there is no ambiguity what needs to be done to reproduce the stuff.

floswald commented on chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd:
````diff
@@ -0,0 +1,1088 @@
+  theme_bw(base_size = 20) +
+  theme(legend.position = "top")
+```
+]
````
that bracket needs to come after the last paragraph. you can't have the `--` in the pull-right.

gustavekenedi replied:
should be a `pull-right` my bad. interestingly it shows in the right sidebar.

floswald commented on chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd:
```diff
@@ -0,0 +1,1088 @@
+ 
+--
+
+* ***Question***: Suppose we had set $\alpha = 0.01 = 1\%$, would we have rejected the absence of discrimination at this level? 
```
this kind of slide is too packed for my taste. there is too much stuff on it. I would try to put less or split them.

gustavekenedi replied:
i will take out the second line because it's really a repetition of the graph and we say it orally.

floswald commented on chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd:
```diff
@@ -0,0 +1,1088 @@
+
+--
+
+* ***Question***: Is the estimated effect statistically significantly different from 0?
```
"why 0?" is a good question of a student at this point. the value `0` here is very deep - it relates back to what a model is, which variables should be in the model. be sure to explain that here - or defer it and just say *is statistically different / greater / smaller than some value `z`*. I think i'd prefer to keep the zero out of it at this stage.

gustavekenedi replied:
i just put it there to give a sneak preview as to what we were going to be discussing in the next lecture. the idea is actually not to answer it now. but i'd be happy to change 0 to some value `z`.

floswald reviewed (changes requested):
great work guys. some comments enclosed

gustavekenedi committed:
change pull-left to pull-right and delete one bullet point on slide 37

gustavekenedi committed:
rectify imprecision in def of CLT in slide 2

gustavekenedi commented:
Thanks very much for your comments Florian!

I've made a few changes, and will make the rest of the changes once I get your replies :)

gustavekenedi committed:
address Florian's comments:
- slide 2: take out one bullet point
- slide 7: add library(tidyverse)
- slide 30: change significantly different from 0 to difference from some value z
- add appendix containing the code to generate the null distribution.

gustavekenedi committed:
commit html

gustavekenedi committed:
add pdf

gustavekenedi committed:
commit pasta_percentile_cis.rds

gustavekenedi commented:
I've made the requested changes and added an appendix slide providing the code to generate the null distribution in case some students want to redo the exercise at home.
